include/NovelRT/Maths/Utilities.h

```diff
@@ -82,6 +110,11 @@ namespace NovelRT::Maths
             }
         }
 
+        /**
+         * @brief Count the preceeding zeroes of a given 64 bits unsigned integer
+         * @param value Then given integer to count the preceeding zeroes
+         * @return Number of zeroes proceeding the given value
+         */
```

Comment:
Just grammar changes, but much like the 32 bit version I'm unsure if I've 100% understood what this was trying to convey.
```suggestion
        /**
         * @brief Counts the preceeding zeroes of a given uint64_t,
         * @param value The given integer to count the preceeding zeroes on.
         * @return Number of zeroes preceeding the given value.
         */
```